api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java

```diff
@@ -1111,7 +1113,8 @@ public static CohortDefinition getMQ13DenB4_P4(
       int adultoSeguimentoEncounterType,
       int hivViralLoadConcept,
       int yesConcept,
-      int pregnantConcept) {
+      int pregnantConcept,
+      boolean greaterThan50) {
```

Comment:
You should change the parameter type to integer, so that the client can set the amount of viral load that needs to be compared in the parameter. This will make the method reusable and avoid adding another if statement in case of a new viral load value.

Reply:
Recomendation implemented.